CHANGELOG.md

```diff
@@ -18,6 +18,14 @@ Please follow the recommendations outlined at [keepachangelog.com](http://keepac
 ### [Unreleased]
 Changes since the last non-beta release.
 
+### [14.0.2] - 2024-05-20
+
+#### Removed
+- Dropped Shakapacker v6 support [PR 1622](https://github.com/shakacode/react_on_rails/pull/1622) by [adriangohjw](https://github.com/adriangohjw)
```

Comment:
This would be a breaking change and require going to v15.

could we still support shakapacker v6? 

@Judahmeek ?

Reply:
@justin808 yes, creating a POC PR for that